mods/persistence/datums/mind/mind.dm

```diff
@@ -0,0 +1,33 @@
+/datum/mind
+	// This is a unique UID that will forever identify this mind.
+	// No two minds are ever the same, and this ID will always identify 'this character'.
+	var/unique_id
+	var/age = 0// How old the mob's mind is in years.
+	var/philotic_damage = 0
+
+/datum/mind/New()
+	. = ..()
+	unique_id = "[sequential_id("/datum/mind")]"
+
+/datum/mind/proc/show_valid_respawns(var/ckey)
+	var/list/valid_clone_pods = list()
+	for(var/network_id in GLOB.computer_networks)
+		var/datum/computer_network/network = GLOB.computer_networks[network_id]
+		for(var/datum/extension/network_device/cloning_pod/CP in network.devices)
+			if(CP.is_valid_respawn(unique_id))
+				valid_clone_pods["Backup (DATEGOHERE) - (AREA NAME HERE)"] = CP
+	
+	if(valid_clone_pods.len)
+		var/clone_pod_tag = input(usr, "Choose a cloning pod to awaken at:", "Select Cloning Pod") as null|anything in valid_clone_pods
+		if(clone_pod_tag)
+			var/datum/extension/network_device/cloning_pod/CP = valid_clone_pods[clone_pod_tag]
+			if(!CP.is_valid_respawn(unique_id))
+				to_chat(usr, SPAN_WARNING("That cloning pod has become unavailable. Please choose a new cloning pod."))
+				return show_valid_respawns(ckey)
+			CP.create_character(src, ckey)
+			philotic_damage += rand(3, 15)
+			if(philotic_damage > 75)
+				to_chat(usr, SPAN_WARNING("Colors seem to lack the vibrance they used to have. It would be easy to just go to sleep and never wake up."))
```

Comment:
You could literally add a client color here to reduce the vibrance of colors. A neat flavor thing

Reply:
Oh shit, what would that look like?